saleor/graphql/discount/mutations/promotion/promotion_update.py

```diff
@@ -142,3 +145,22 @@ def send_promotion_toggle_webhook(
             cls.call_event(event, instance)
             instance.last_notification_scheduled_at = now
             instance.save(update_fields=["last_notification_scheduled_at"])
+            if event == manager.promotion_started:
+                return "started"
+            if event == manager.promotion_ended:
+                return "ended"
+        return None
+
+    @classmethod
+    def save_events(
+        cls,
+        info: ResolveInfo,
+        instance: models.Promotion,
+        sent_webhook_type: Optional[str],
```

Comment:
Also to be precise. the name `sent_webhook_type` is not correct. You don't directly trigger the webhook. You trigger the event on PluginsManager side. WebhookPlugin is one of many plugins that could be subscribed to the event triggered by this method.